docs/tutorials/jokes.md

```diff
@@ -6351,10 +6351,118 @@ This will build the docker image and deploy it on Fly in the region you selected
 
 Great! We're done and you made someone's day! Success!
 
-Your app is now live at `https://<your-app-name>.fly.dev`! You can find that URL in your fly account online as well: [fly.io/apps](https://fly.io/apps).
+Your app is now live at `https://<your-app-name>.fly.dev`! You can find that URL in your fly account online as well: [fly.io/apps][fly-io-apps].
 
 Any time you make a change, simply run `fly deploy` again to redeploy.
 
 ## Conclusion
 
-Phew! And there we have it. If you made it through this whole thing then I'm really impressed ([tweet your success](https://twitter.com/intent/tweet?text=I%20went%20through%20the%20whole%20remix.run%20jokes%20tutorial!%20%F0%9F%92%BF%20And%20now%20I%20love%20@remix_run!&url=https://remix.run/tutorials/jokes))! There's a lot to Remix and we've only gotten you started. Good luck on the rest of your Remix journey!
+Phew! And there we have it. If you made it through this whole thing then I'm really impressed ([tweet your success][tweet-your-success])! There's a lot to Remix and we've only gotten you started. Good luck on the rest of your Remix journey!
+
+[remix-jokes]: https://remix-jokes.lol
+[mdn]: https://developer.mozilla.org/en-US/
+[prisma]: https://www.prisma.io/
+[code-sandbox]: https://codesandbox.com/
+[node-js]: https://nodejs.org
+[npm]: https://www.npmjs.com
+[vs-code]: https://code.visualstudio.com/
+[fly-io]: https://fly.io
+[java-script-to-know-for-react]: https://kentcdodds.com/blog/javascript-to-know-for-react
+[the-beginner-s-guide-to-react]: https://kcd.im/beginner-react
+[the-http-api]: https://developer.mozilla.org/en-US/docs/Web/HTTP
+[the-basic-example]: https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/basic
+[express]: https://expressjs.com/
+[hydrate]: https://reactjs.org/docs/react-dom.html#hydrate
+[http-localhost-3000]: http://localhost:3000
+[bare-bones-hello-world-app]: /jokes-tutorial/img/bare-bones.png
+[remix-config-js]: ../api/conventions#remixconfigjs
+[route-module]: ../api/conventions#route-module-api
+[the-route-filename-convention]: ../api/conventions#route-filenames
+[react-router]: https://reactrouter.com/
+[a-greeting-from-the-index-route]: /jokes-tutorial/img/index-route-greeting.png
+[jokes]: http://localhost:3000/jokes
+[a-random-joke-on-the-jokes-page-i-was-wondering-why-the-frisbee-was-getting-bigger-then-it-hit-me]: /jokes-tutorial/img/random-joke.png
+[jokes-new]: http://localhost:3000/jokes/new
+[a-new-joke-form]: /jokes-tutorial/img/new-joke.png
+[jokes-anything-you-want]: http://localhost:3000/jokes/hippos
+[a-new-joke-form-2]: /jokes-tutorial/img/param-route.png
+[links]: ../api/conventions#links
+[links-2]: ../api/remix#link
+[http-localhost-3000-2]: http://localhost:3000/
```

Comment:
This is the same as `http-localhost-3000`, except it has a trailing slash

Reply:
which makes it different 🥴